src/main/kotlin/no/nav/syfo/melding/api/MeldingSystemApi.kt

```diff
@@ -0,0 +1,36 @@
+package no.nav.syfo.melding.api
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.syfo.application.api.access.APIConsumerAccessService
+import no.nav.syfo.melding.MeldingService
+import no.nav.syfo.util.getBearerHeader
+
+const val meldingSystemApiBasePath = "/api/system/v1/melding"
```

Comment:
Hvis vi skal følge rest konvensjon så lurer jeg på om det blir riktigere med flertall på meldinger her😊
```suggestion
const val meldingSystemApiBasePath = "/api/system/v1/meldinger"
```
([Mer info her](https://restfulapi.net/resource-naming/) om noen er interessert)

Reply:
Ja, riktig, fikser :+1: